astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,196 @@
+import DockerRequirements from 'src/components/quickstarts/DockerRequirements.astro'
+import DockerInstallAndStart from 'src/components/quickstarts/DockerInstallAndStart.astro'
+import GitPodStart from 'src/components/quickstarts/GitPodStart.astro'
+
+To get started with FusionAuth, you need to do three things:
+
+* Launch FusionAuth
+* Delegate authentication
+* Add functionality to your application
+
+<StartDiagram alt="Launch, then delegate, then add functionality." />
+
+Let's walk through an example where you'll add a Login With Google button to an application with no code changes required.
+
+## Prerequisites
+
+* git
+* Docker (if you want to follow the Docker instructions)
+* Github account (if you choose to use GitPod instead)
+
+## Launch FusionAuth
+
+FusionAuth is downloadable software, so you need to run it somewhere. For this guide, you can either
+
+* Use Docker to install and run FusionAuth and the application
+* Use GitPod to run FusionAuth in the cloud in your GitPod account, with no software install needed.
+
+Expand the section you want to follow and get started.
+
+<details>
+<summary>Docker</summary>
+
+<DockerRequirements />
+<DockerInstallAndStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+
+<details>
+<summary>GitPod</summary>
+
+<GitPodStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+
+</details>
+
+### Delegate Authentication
+
+The next step is to delegate authentication for your application, and verify that the change is applied.  For this you will:
+* Add the admin@example.com user to the application
+* Logout from the administrative user
+* Go to the test application
+* Login as the administrative user
+
+### Step 1. Bring up the administrative interface
+
+In the previous step, you logged in to the system as an admin user.  Go back to that browser screen, which should be showing the administrative interface for FusionAuth.
+
+### Step 2. Choose the correct user
+
+* Navigate to "Users" in the left-hand sidebar (you may need to use the hamburger icon at the top left of the screen to see all options)
+* Choose the "admin@example.com" user (Dinesh Chugtai)
+* Choose "Manage" from the list of actions
+
+### Step 3. Add registration to the application
+
+* Scroll down to the bottom of the user page and click "Add registration"
+* Choose the "Start Here" application (it may already be selected)
+* Click the blue save icon at the top of the screen
+
+### Step 4.  Log out from the administration interface
+
+* If the left navigation bar is not visible, click the hamburger icon at the top left to see the options.
+* At the top of the left navigation bar, click "Logout"
+
+### Step 5. Startup the application
+
+In your shell, type the following to start up the application:
+
+```
+cd app
+npm install
+npm run dev
+```
+
+### Step 6. Open the application
+
+For gitpod:
```

Comment:
Should we have these as accordions for consistency?

Reply:
Added accordions for Delegate steps